src/types/chain.ts

```diff
@@ -0,0 +1,35 @@
+export type EVMChainID =
+  //polygon
+  | "137"
+  | "80001"
+  //ethereum
+  | "1"
+  | "5"
+  //binance
+  | "56"
+  | "97";
+
+export type CosmosChainID = "juno-1" | "uni-6";
+//would remove this type once terra tooling is unified to that of cosmos (keplr)
+export type TerraChainID = "phoenix-1" | "pisco-1";
+
+export type ChainID = EVMChainID | CosmosChainID | TerraChainID;
+type NativeAtomicUnit = string; //ujunox uluna
+type IBCDenom = string;
+
+export type Chain = {
+  id: ChainID;
```

Comment:
as we now control the chains user selects from, we could type this to a finite set. However, `wallet.chainID` is still `string` as we can't possibly know if user's wallet is connected to our supported chains without checking